README.md

```diff
@@ -35,25 +34,44 @@ You need [Google Chrome](https://www.google.com/chrome/browser/desktop/). *not t
 
 <img src="https://github.com/isaacbernat/netflix-to-srt/blob/master/chrome_console.png?raw=true" alt="Chrome console screenshot" width="557px" height="607px">
 
-### From Netflix: method 3
+## Method 3
 The information is extracted from [this post](http://forum.opensubtitles.org/viewtopic.php?t=15141).
 
 You need FireFox and AdblockPlus Add-On. *not tested on other browsers*
 - Start Netflix and your movie/episode (stream is active!)
 - Start AdblockPlus, open blockable items
-- Search: dfxp *(e.g. >> #.nflximg.com/#/#/########.dfxp?v=1&e=#########&t=######_#####&random=1234567890)*
+- Search: dfxp *(e.g. `>> #.nflximg.com/#/#/########.dfxp?v=1&e=#########&t=######_#####&random=1234567890`)*
 - open the dfxp in a new window
 - Save as
 
-### From YouTube
+# 2. YouTube-To-SRT
 - Install [youtube-dl](https://github.com/ytdl-org/youtube-dl) (available for Windows, Mac and Linux)
 - Download subs from the YouTube URL you like e.g. `youtube-dl --all-subs "https://www.youtube.com/watch?v=VHNfvFOBC0A"`
 - Subtitles should be downloaded in the same folder were the command was ran. E.g. `NameOfTheVideo VHNfvFOBC0A.ca.vtt, NameOfTheVideo VHNfvFOBC0A.tlh.vtt`
 - If you are missing a language, check that it's actually available. E.g. `youtube-dl --list-subs "https://www.youtube.com/watch?v=VHNfvFOBC0A"`
+> **Note:** you can also use [youtube-dlp](https://github.com/yt-dlp/yt-dlp-wiki/blob/master/Installation.md) which is the fork version of [youtube-dl](https://github.com/ytdl-org/youtube-dl) by [Snap](https://snapcraft.io/yt-dlp)
+ - Install [yt-dlp](https://github.com/yt-dlp/yt-dlp) using Snap:
+ ```bash
+ sudo snap install --edge yt-dlp
+ ```
+ - To download subtitles in SRT format from a YouTube URL, use:
+ ```bash
+  yt-dlp --skip-download --write-auto-subs --convert-subs srt --sub-lang "en" "https://youtu.be/cVsyJvxX48A" 
+ ```
+ This command downloads only the subtitles in SRT format.
+ - If you want to download the subtitles in VTT format, you can use:
+ ```bash
+ yt-dlp --skip-download --write-auto-subs --sub-lang "en" "https://youtu.be/cVsyJvxX48A"
+ ```
+ - To download the video with audio and subtitles, simply omit the --skip-download option:
+ ```bash
+ yt-dlp --write-auto-subs --sub-lang "en" "https://youtu.be/cVsyJvxX48A"
+ ```
+> **Note:** If you don't enable subtitles in the YouTube video (captions), then the command will not work as expected. Make sure to enable subtitles before running the command.
 
-## Convert them into .srt
+## Clone Repo & Run Files
 - [Get python](https://www.python.org/downloads/) (tested under python 2.7, 3.3 and newer). *If you have mac or linux you may skip this step*
-- Clone this repository or [download `to_srt.py`](https://raw.githubusercontent.com/isaacbernat/netflix-to-srt/master/to_srt.py)
+- Clone this repository or [download the ZIP file](https://github.com/isaacbernat/netflix-to-srt/archive/refs/heads/master.zip).
```

Comment:
Ahh! I just neglected it because it might be easy for me to download like that, so I thought it would be easy for others too. just a perspective; Sorry, I forgot about it; I will correct it right now.